script.js

```diff
@@ -0,0 +1,11 @@
+let markMass = 70;
```

Comment:
Why using let here?

Reply:
My bad, realized i should have been using const on the second challenge and fixed the declarations in all the PRs except for this one.